scripts/disable_networking.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Seems like this script and `enable_networking.sh` could just be run directly from `bandwidth_throttling.sh` as different commands.